content/6_Advanced/Vectorization.mdx

```diff
@@ -133,3 +133,265 @@ instead of faster (ex. see
 <!-- Anyways, keep in mind that these rarely make a difference in competitive programming. -->
 
 <!-- <IncompleteSection /> -->
+
+## Example - Ligatures
+
+[Problem](https://open.kattis.com/problems/ligatures?editresubmit=13798177)
+We have yet to find a problem on this topic. In the meantime, use this.
```

Comment:
wdym by this

Reply:
The module states that this concept has not appeared in a problem yet. The content author who brought up this issue linked an example problem that wasn't from USACO, so I wasn't sure how to format this section.

Reply:
this could probably be omitted - the module already mentions this 👍